### DISCUSSION TITLE
[Book] The open source way

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#30] [资料分享]
![image](https://user-images.githubusercontent.com/15010826/158276777-0d87cbec-53f4-4f42-bb59-e2173f3aea38.png)

推荐这本好书，很好的开源社区贡献、管理、运营方面的介绍。The open source way is a way of thinking about how people collaborate within a community to achieve common goals and interests.

https://www.theopensourceway.org/

[the_open_source_way-guidebook-2.0.pdf](https://github.com/kaiyuanshe/ONES/files/8249029/the_open_source_way-guidebook-2.0.pdf)

**Comment by wangleo61** [comment]
谢谢王老师推荐
